content/maintainers-guide/processes/release/prestashop/create-version-branch.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Create a new version branch
+weight: 20
+aliases:
+  - /maintainers-guide/releasing-prestashop/create-version-branch/
+---
+
+# Create a new branch for a new major/minor version.
+
+
+## 1) Create the new branch and push it to the main repo
+
+## 2) Open PRs based on the new branch and modify those files :
+
+There are multiples repo to update.
+
+We'll list the file we modified for the creation of the 9.0.x branch.
+
+Changed files :
+
+PrestaShop/PrestaShop :
+
+- .github/workflows/cron_js_routing.yml
+- .github/workflows/cron_nightly_build.yml
+- .github/workflows/cron_nightly_tests_9.0.x_mariadb.yml ( created )
+- .github/workflows/cron_nightly_tests_9.0.x_mysql.yml ( created )
+- .github/workflows/cron_nightly_tests_reports.yml
+- .github/workflows/cron_nightly_tests_reusable.yml
+- .github/workflows/cron_php_update_modules.yml
+- .github/PULL_REQUEST_TEMPLATE.md
+
+PrestaShop/test-scenarios :
+
+- .github/workflows/gh-pages.yml
+- config.json
+
+PrestaShop/presthubot :
+
+- src/App/Command/SlackNotifierCommand.php
+
+
+PrestaShop/prestashop.github.io : ( no PR needed this time )
```

Comment:
```suggestion
`PrestaShop/prestashop.github.io`: (you can commit the changes directly, no need to create a pull request)
```

Reply:
That's not what I meant. I meant that there was on changes needed on this repo. They had already been done.

Reply:
I'm not sure I understand, then, from what you're writing, this information is irrelevant. The change wasn't required this time, but it might be in the future, right? Do I understand correctly? Remember that you don't document what happened in v9, you document something for the future use 👍🏻